src/elements/file-selector/readme.md

```diff
@@ -93,24 +96,31 @@ It's suggested to have a different value for each variant, e.g.:
+| `files`              | -                     | public  | `File[]`                    |             | The list of selected files.                                              |
+| `form`               | -                     | public  | `HTMLFormElement \| null`   |             | Returns the form owner of the internals of the target element.           |
 | `multiple`           | `multiple`            | public  | `boolean`                   | `false`     | Whether more than one file can be selected.                              |
 | `multipleMode`       | `multiple-mode`       | public  | `'default' \| 'persistent'` | `'default'` | Whether the newly added files should override the previously added ones. |
+| `name`               | `name`                | public  | `string`                    |             | Name of the form element. Will be read from name attribute.              |
 | `size`               | `size`                | public  | `'s' \| 'm'`                | `'m'`       | Size variant, either s or m.                                             |
 | `titleContent`       | `title-content`       | public  | `string \| undefined`       |             | The title displayed in `dropzone` variant.                               |
+| `value`              | `value`               | public  | `V \| null`                 | `null`      | Value of the form element.                                               |
 | `variant`            | `variant`             | public  | `'default' \| 'dropzone'`   | `'default'` | Whether the component has a dropzone area or not.                        |
 
 ## Methods
 
-| Name       | Privacy | Description | Parameters | Return   | Inherited From |
-| ---------- | ------- | ----------- | ---------- | -------- | -------------- |
-| `getFiles` | public  |             |            | `File[]` |                |
+| Name                       | Privacy | Description | Parameters                                                    | Return   | Inherited From |
+| -------------------------- | ------- | ----------- | ------------------------------------------------------------- | -------- | -------------- |
+| `formResetCallback`        | public  |             |                                                               | `void`   |                |
+| `formStateRestoreCallback` | public  |             | `state: FormRestoreState \| null, _reason: FormRestoreReason` | `void`   |                |
+| `getFiles`                 | public  |             |                                                               | `File[]` |                |
 
 ## Events
 
 | Name          | Type                  | Description                                                | Inherited From |
 | ------------- | --------------------- | ---------------------------------------------------------- | -------------- |
+| `change`      | `Event`               |                                                            |                |
```

Comment:
not sure whether documentation should be added?

Reply:
I had no idea that this was enough for the event to be added to the doc :|
![image](https://github.com/user-attachments/assets/b5b1fcef-a704-461a-bc01-8ab83a361ea8)

Reply:
oh yeah it does :D. btw. we have long-term plans to also document change and input events on every form component, so it's not wrong at all ;-)

Reply:
Inspired by MDN, I added a doc which should be fine for every form element